app/models/user.rb

```diff
@@ -325,29 +328,26 @@ def perform_export!
     else
       update exporting: false
     end
-  rescue => error
-    logger.error "Unexpected error while exporting user '#{username}': #{error.class}: #{error.message}\n" \
-                 "#{error.backtrace.first(15).join("\n")}"
+  rescue StandardError => e
+    logger.error "Unexpected error while exporting user '#{username}': #{e.class}: #{e.message}\n" \
```

Comment:
- Unexpected error while exporting data for '#{username}':
This mirrors the error message for photos.